src/components/shared/Extensions/Tabs/Tabs.tsx

```diff
@@ -25,7 +25,7 @@ const Tabs: FC<PropsWithChildren<TabsProps>> = ({
         activeTab={activeTab}
         onTabClick={onTabClick}
         tabsUpperContainerClassName={clsx(
-          'before:content-[" "] relative flex w-full items-center p-0 font-semibold before:absolute before:bottom-0 before:left-0 before:block before:h-px before:w-full before:bg-gray-200',
+          'before:content-[" "] relative flex w-full items-start p-0 font-semibold before:absolute before:bottom-0 before:left-0 before:block before:h-px before:w-full before:bg-gray-200',
           upperContainerClassName,
         )}
         leftNavBtnClassName="absolute top-[50%] translate-y-[-50%] left-0 z-base"
```

Comment:
I think you need to use only `z-base` also for `leftNavBtnClassName`

Reply:
No actually we need to remove everything, apart from z-base 😅  I'll push 2 secs

Reply:
Pushed! 🚀